src/q7/radiation-ELEC2795/exam/2019/Janvier/All/radiation-ELEC2795-exam-2019-Janvier-All.tex

```diff
@@ -177,6 +290,20 @@ \section{(4,5 points)}
 
 \end{enumerate}
 
-\nosolution
+\begin{solution}
+ \begin{enumerate}
+  \item c. The Fourier transform of a complex exponential (auto-correlation function for a Ricean distribution) is a shifted delta.
+  \item b. $$K>\frac{1}{3} ((C/I) \, K_I)^{2/n} = 5.1$$
```

Comment:
I'd use `\[...\]`, but the difference is negligible (something with spacing, iirc).

Reply:
```suggestion
  \item b. \[ K>\frac{1}{3} ((C/I) \, K_I)^{2/n} = 5.1 \]
```